provider/httprequest_provider.py

```diff
@@ -227,8 +227,10 @@ def _get_img_by_size(self, img, image_size: str = '') -> str:
         if image_size != 'm' and image_size != 'l':
             image_size = 's'
 
-        # 从url解析图片id
+        # 从url解析图片id，空白默认图没id，不处理
         match = re.search(r"/p(\d+?)\.", img)
+        if not match:
+        	return img
```

Comment:
```suggestion
        if not match:
            return img
```

格式化不要使用 tab 而是 space

Reply:
已提交修改